benchmarks/operator_benchmark/pt/fill_test.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
```

Comment:
Are these imports needed? (e.g., I don't see print used)

Reply:
That was just to make internal linter happy. It's ok to remove them.